package.json

```diff
@@ -8,7 +8,8 @@
     "prepare": "esno scripts/prepare.ts",
     "taze": "taze minor -wIr && pnpm -r --parallel run update-post",
     "bench": "npm -C bench run bench",
-    "build": "rimraf packages/*/dist --glob && esno scripts/copy-files.ts && pnpm -r --filter=./packages-*/* run build && pnpm -r run build-post && esno scripts/postbuild.ts",
+    "build": "rimraf packages/*/dist --glob && esno scripts/copy-files.ts && pnpm -r --filter=./packages-*/* run build && pnpm -r run build-post",
```

Comment:
We already have clean options set to true on unbuild config, is there any reason to add rimraf to build script?